frameworks/digital-outcomes-and-specialists-5/manifests/display_brief.yml

```
@@ -59,6 +59,7 @@
     - niceToHaveRequirementsParticipants
 -
   name: How suppliers will be evaluated
+  summary_page_description: "All suppliers will be asked to provide a {% if lotSlug == 'digital-specialists' %}work history{% else %}written proposal{% endif %}."
```

I've been testing in flask shell, I think it should be `lot` instead of `lotSlug`

```suggestion
  summary_page_description: "All suppliers will be asked to provide a {% if lot == 'digital-specialists' %}work history{% else %}written proposal{% endif %}."
```